A62-pick-first.md

```diff
@@ -123,22 +146,40 @@ addresses it receives before it starts to attempt to connect.
 }
 ```
 
-gRPC clients that receive `pick_first` configuration with this field set, but do
-not support this configuration should continue to work and their behavior should
-remain unchanged. Implementations that support the new field and receive
-configuration with `shuffleAddressList` set to `true`, should shuffle the
-received address list at random before attempting to connect to them.
+gRPC clients that receive `pick_first` configuration with this field set, should
```

Comment:
Suggest wording this as follows:

"""
In a gRPC implementation that supports this feature, when the `shuffleAddressList` option is enabled, the `pick_first` policy will randomly shuffle the order of the addresses before attempting to connect to them.

Note that existing gRPC implementations that do not support this feature will ignore this field, so their behavior will remain unchanged.
"""

Reply:
Done.